app/resources/nested_resources/file_metadata.rb

```diff
@@ -42,6 +42,10 @@ def derivative?
     use.include?(Valkyrie::Vocab::PCDMUse.ServiceFile)
   end
 
+  def derivative_partial?
+    use.include?(Valkyrie::Vocab::PCDMUse.ServiceFilePartial)
```

Comment:
Does this exist in valkyrie?  I don't see it in https://github.com/samvera-labs/valkyrie/blob/master/lib/valkyrie/vocab/pcdm_use.rb

Reply:
No, it's totally made up, because nothing else fit. Works because that vocab's not a strict one.